silkworm/core/execution/call_tracer.cpp

```diff
@@ -18,11 +18,58 @@
 
 #include <evmc/hex.hpp>
 #include <evmc/instructions.h>
+#include <evmone/baseline_instruction_table.hpp>
 #include <evmone/execution_state.hpp>
 #include <evmone/instructions.hpp>
 
 #include <silkworm/core/types/address.hpp>
 
+using namespace evmone;
+using namespace evmone::baseline;
+
+// The following check_requirements function has been temporarily copied from evmone because it is not exported.
+// We need to ask evmone for it to be exported or for a tracing interface extension (e.g. on_instruction_end?).
+namespace {
+template <Opcode Op>
+inline evmc_status_code check_requirements(const CostTable& cost_table, int64_t& gas_left,
```

Comment:
```suggestion
[[deprecated("Temporary fix, await tracing interface extension")]]
inline evmc_status_code check_requirements(const CostTable& cost_table, int64_t& gas_left,
```

Reply:
I tried to include this `deprecated` attribute but then the build fails because we do not accept any deprecated code call, I forgot about it. So, given that there's a comment clarifying the presence of the `check_requirements` function here, I decided not to include the attribute in order not to make this translation unit too cluttered.